src/client/app/graph/card/card.component.ts

```diff
@@ -109,21 +106,23 @@ export class CardComponent implements OnInit {
       const newCardPosition =
         this.canvasService.viewportCoordToCanvasCoord(this.position);
       this.card.position = {x: newCardPosition.x, y: newCardPosition.y};
-      // TODO: move all associated arrows too
-      let inArrow = this.card.incomingArrow;
-      if (inArrow) {
-        inArrow.tipPosition = {x: newCardPosition.x, y: newCardPosition.y + 40};
-        if (inArrow.fromCardId === null) {
-          inArrow.tailPosition = {x: newCardPosition.x - 50, y: newCardPosition.y + 40};
+      // Move all associated arrows too
+      this.card.incomingArrows.asArray().forEach((inArrow: Arrow) => {
+        if (inArrow) {
+          inArrow.tipPosition = {x: newCardPosition.x, y: newCardPosition.y + this.card.size.height/2};
+          if (inArrow.fromCard === null) {
+            inArrow.tailPosition = {x: newCardPosition.x - 50, y: newCardPosition.y + this.card.size.height/2};
+          }
         }
-      }
-      let outArrow = this.card.outgoingArrow;
-      if (outArrow) {
-        outArrow.tailPosition = {x: newCardPosition.x + 60, y: newCardPosition.y + 40};
-        if (outArrow.toCardId === null) {
-          outArrow.tipPosition = {x: newCardPosition.x + 60 + 50, y: newCardPosition.y + 40};
+      });
+      this.card.outgoingArrows.asArray().forEach((outArrow: Arrow) => {
+        if (outArrow) {
+          outArrow.tailPosition = { x: newCardPosition.x + this.card.size.width, y: newCardPosition.y + this.card.size.height/2 };
+          if (outArrow.toCard === null) {
+            outArrow.tipPosition = { x: newCardPosition.x + this.card.size.width + 50, y: newCardPosition.y + this.card.size.height/2 };
```

Comment:
Let's follow 80 character line limits throughout?

Reply:
Done for this file. Though this gonna be hard to maintain for the whole codebase without a formatting tool that enforces this. We should probably set one up.